### PR TITLE
maint/add-contributing-md

### PR DESCRIPTION
**Pull request by kylog**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# How to contribute
+
+Third-party patches are essential for keeping facter great. We simply can't
```
Comment (kylog) [inline CONTRIBUTING.md]: I started to rename 'facter' to 'native facter' or 'cfacter' but those both seem like mouthfuls, and before long the executable will just be 'facter'.  So I let it be :)

**Comment by MikaelSmith** [comment]
Is this copied straight from Facter?

**Comment by kylog** [comment]
Yes, but with three minor diffs:
* added a paragraph about the schema location being odd for now
* updated the sample commit message to be jira style not redmine style
* updated the jira link to be CFACT specific

**Comment by peterhuene** [comment]
I'm hoping the reference to the schema file may be relevant soon, as I plan on tackling that next after I'm done with Catch.  It'd be nice from a test standpoint as well as a place we can start documenting the facts.

**Comment by MikaelSmith** [comment]
:+1:

**Pull request merged**